cxotime/tests/test_utils.py

```diff
@@ -0,0 +1,89 @@
+import astropy.units as u
+import numpy as np
+import pytest
+
+from cxotime import CxoTime, get_range_in_chunks
+
+
+@pytest.mark.parametrize(
+    "start, stop, dt_max, expected_len, expected_values",
+    [
```

Comment:
The ruff formatting made this ugly enough that maybe this should be de-parameterized.